taxonomies/ingredients.txt

```diff
@@ -36895,7 +36943,7 @@ fi:ananaspyree
 en:pineapple juice
 da:Ananasjuice
 de:Ananassaft
-es:Zumo de piña
+es:Zumo de piña, jugo de piña
```

Comment:
instead of adding jugo de XYZ as synonyms to all zumo de XYZ, we can just declare jugo to be a synonym of zumu, at the top of the ingredients.txt file

Reply:
ok where is this done? - I'm still trying to work out the structure of the ingredients.txt file..

Reply:
ok, spotted it now